### ISSUE TITLE
Feature request: Recent file list

### ISSUE THREAD

**Issue by cechurat** [mb1986/rm-hacks#96]
First of all, let me thank you @mb1986  for using my rM again with more and more pleasure.

There is one handy thing I'm missing  - display the **recent file list**.
Here is a link to ddvk's implementation of this feature: https://github.com/ddvk/remarkable-hacks/blob/master/docs/images/screenshot_recent_files.png

_"two finger swipe up or long press home button for recent files"_

**Comment by PepikVaio** [comment]
reMarkable has this function natively 2 fingers from the top edge.
I don't think it's worth a duplicate.

**Comment by martinhess-at** [comment]
Hi there, the rM overview can't compete with the ugly but very efficient filelist of ddvks. So for me its not a question of duplicate, its totally different and better. Cheers Martin

**Comment by dreadful-brain** [comment]
> Hi there, the rM overview can't compete with the ugly but very efficient filelist of ddvks. So for me its not a question of duplicate, its totally different and better. Cheers Martin

I agree 100 percent, the native current files are simply not usable if you work with more than 4 files on a regular basis.

**Comment by PepikVaio** [comment]
OK, they don't work with more than 4 files at a time.

Actually, I'm exhibiting with 2 and that's an awesome feature 2 fingers from the bottom :)

**Comment by Tridens92** [comment]
I frequently work back and forth between about eight files every day, and would love to see this feature integrated.  Thanks!

**Comment by rudy0ne** [comment]
I would love an option for the Favorites list with the same gesture. I don't like how on the rM2 you basically have to scrape your finger tips against the top edge with the native Recents/Favorites gesture.

**Comment by PepikVaio** [comment]
OK, let's make this feature nicer mb1986 style and not DDVK style.
